controllers/applications.ts

```diff
@@ -0,0 +1,134 @@
+import { addLog } from "../models/logs";
+const { logType } = require("../constants/logs");
+import { CustomRequest, CustomResponse } from "../types/global";
+const { INTERNAL_SERVER_ERROR } = require("../constants/errorMessages");
+const ApplicationModel = require("../models/applications");
+const { API_RESPONSE_MESSAGES } = require("../constants/application");
+const { getUserApplicationObject } = require("../utils/application");
+const admin = require("firebase-admin");
+
+const getAllOrUserApplication = async (req: CustomRequest, res: CustomResponse): Promise<any> => {
+  try {
+    const { userId, status, next, size } = req.query;
+    const limit = Number(size) || 25;
+    let nextPageUrl = null;
+
+    if (userId) {
+      const applications = await ApplicationModel.getUserApplications(userId);
+
+      if (!applications.length) return res.boom.notFound("User application not found");
```

Comment:
we should return the response in same form as below with an empty array with status 200 i.e we can remove this line.